app/assets/stylesheets/themes/light/constants.less

```diff
@@ -26,10 +26,6 @@
 @sidebar-width: 300px;
 @container-max-width: 970px;
 
-.card-dark {
-    border: 1px solid @border-light-color;
-}
-
 .card-light {
-    box-shadow: 0 2px 6px -2px @border-light-color;
+    border: 1px solid @border-light-color;
 }
```

Comment:
I dunno if I would remove card-dark completely rather than make it an empty mixin. Allows potential extendability if we choose to give all card-darks something for the light theme...

Would need other opinions here though.